### PR TITLE
 #185218908 ch-setup sass 

### PR DESCRIPTION
**Pull request by irakozejules**
## What does this PR do?
- This PR set the sass to be used in the project

## Description of Task to be completed?

- Install and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client

## How should this be manually tested?

- Clone the repository using `git clone` [link](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn)

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

## What are the relevant pivotal tracker/Trello stories?

[#185218908](https://www.pivotaltracker.com/story/show/185218908)

### CODE REVIEW

**Review by jcharles921 — APPROVED** [review]
Good job

**Review by bergad37 — APPROVED** [review]
Awesome!

File: src/home.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const Home = () => {
```
Comment (nishimweprince) [inline src/home.jsx:1]: To maintain a consistent folder structure, please move the homepage file in a `/pages` directory.

**Review by ReneNtabana — APPROVED** [review]

**Review by nishimweprince — CHANGES_REQUESTED** [review]
Please update the PR message to remove a portion of your commit message at the start. Also, update your commit message referring to this section in our README file: [Linking GitHub With PivotalTracker](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/tree/ch-setup-react-router-185218907#linking-github-with-pivotaltracker)

**Review by irakevine — APPROVED** [review]

**Review by frankflej — APPROVED** [review]
I liked the simple transition you used as an example nice work!

**Review by nishimweprince — APPROVED** [review]

**Pull request merged**